packages/hunspell.rb

```diff
@@ -2,37 +2,20 @@
 
 class Hunspell < Package
   description 'Hunspell is a spell checker and morphological analyzer library'
-  homepage 'http://hunspell.github.io/'
-  version '1.7.0'
+  homepage 'https://hunspell.github.io/'
+  version '1.x'
   license 'MPL-1.1, GPL-2 and LGPL-2.1'
   compatibility 'all'
-  source_url 'https://github.com/hunspell/hunspell/archive/v1.7.0.tar.gz'
-  source_sha256 'bb27b86eb910a8285407cf3ca33b62643a02798cf2eef468c0a74f6c3ee6bc8a'
 
   is_fake
 
-  if ARGV[0] == 'install'
-    puts
-    puts 'Enter your preferred language:'
-    puts '1 = American English'
-    puts '2 = Français'
-    puts '3 = Español'
-    puts '0 = Cancel'
+  def self.preflight
+    options = [
+      { value: 'hunspell_en_us',  description: 'American English' },
+      { value: 'hunspell_fr_fr',  description: 'Français'         },
+      { value: 'hunspell_es_any', description: 'Español'          }
+    ]
 
-    while version = $stdin.gets.chomp
-      case version
-      when '1'
-        depends_on 'hunspell_en_us'
-        break
-      when '2'
-        depends_on 'hunspell_fr_fr'
-        break
-      when '3'
-        depends_on 'hunspell_es_any'
-        break
-      when '0'
-        abort
-      end
-    end
+    depends_on Selector.new(options).show_prompt if ARGV.include?('install')
```

Comment:
Can we wrap the whole block inside `if ARGV.include?('install') ... end`?  That way options is not assigned for every argument.

Reply:
Change made